conda/environments/all_cuda-129_arch-aarch64.yaml

```diff
@@ -56,9 +56,9 @@ dependencies:
 - pylibraft==26.2.*,>=0.0.0a0
 - pyrsistent
 - pytest-cov
-- pytest<8
+- pytest<9.0
```

Comment:
We updated the rest of RAPIDS to PyTest 8 a while ago, and version 9 is out now. Let's try to move this pinning forward.